client/app.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component, Fragment } from 'react'
+import Nav from './components/Nav/Nav'
+import PedalBoards from './components/PedalBoards/PedalBoards'
+
+class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      pedalboards: []
+    }
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3001/pedalboards', {
```

Comment:
This won't work if you deploy the app. Use a relative path instead of a full URL.